blockstore/apps/bundles/store.py

```diff
@@ -121,6 +121,49 @@ def create(cls, bundle_uuid, files, links=None, created_at=None):
             created_at=created_at,
         )
 
+    def _serialized_dep(self, dependency):
+        return {
+            "bundle_uuid": dependency.bundle_uuid,
+            "version": dependency.version,
+            "snapshot_digest": dependency.snapshot_digest.hex(),
+        }
+
+    def _expand_url(self, url, context):
+        """Ensure that the given URL is an absolute URL"""
+        if not url.startswith('http'):
+            url = context['request'].build_absolute_uri(url)
+        return url
+
+    def serialize(self, context):
+        """Serialize a snapshot."""
```

Comment:
Why did you move the serialization code into this `Snapshot` structure class? Seems inconsistent; I'd rather you just move it to a SnapshotSerializer in the serializers file and update SnapshotField to point to that.

Reply:
yep, I separated the log to another serializer